src/renderer/components/side-nav/side-nav.css

```diff
@@ -168,7 +167,10 @@
   }
 
   .sideNav {
+    bottom: 0;
     display: flex;
+    left: 0;
+    position: fixed;
```

Comment:
Can we group the properties for positioning together?

```css
    position: fixed;
    left: 0;
    bottom: 0;

    display: flex;
```
The example code above is like talking to the renderer like this:
- For positioning, I want it to be fixed, 0 spacing to left and bottom
- For display behavior, I want flex

Reply:
Personally I always alphabetically sort CSS properties (with very few exceptions), I find it to be the most reproducible method of property sorting. Implemented your change but I would argue for a future where we set up a linter for the CSS and not have to worry about this ;p